config/settings/test.yml

```diff
@@ -94,13 +94,7 @@ modules_appeals_api:
 bd:
   base_name: staging-api.va.gov/services
 
-# Put a real api_key in the BRD_API_KEY environment variable to record VCR cassettes
-# Be sure to not commit the api key and the cassettes.
-# If used the tunnel and localhost for the base_name, then you don't need an api key
-# If you use localhost, uncomment `ssl: false`.
 brd:
-  # base_name: localhost:4451
-  # ssl: false
```

Comment:
Claims API should be accessing these via the portal as that's how we're interacting with them in our live environments. I believe the intent here was to connect to BRD w/o the need for an API key but we can easily add a config change for VCR to strip the API keys - this would also save us from having to update the `uri:` in the cassettes, so 6 of one, half dozen of the other. It's also best for our devs to have said keys for calls directly to BRD.

Reply:
@stiehlrod  - I think we add 97 and 98 back. we got a bit gung ho removing those. they're meaningful as we don't want to commit an API key to the test.yml on accident. We can write tooling to strip it from cassettes, but not so much the test.yml